dangerfile.ts

```diff
@@ -126,43 +128,28 @@ const checkDependencyChanges: () => void = () => {
   }
 }
 
-// const checkYarnAudit: () => void = () => {
-const checkContributorsJSON: () => void = () => {
-  const packageChanged = allFiles.includes('package.json')
-  if (packageChanged) {
-    danger.git
-      .structuredDiffForFile('package.json')
-      .then((sdiff) => {
-        return sdiff.chunks.every((chunk) => {
-          return chunk.changes
-            .filter((change) => {
-              return change.type === 'add' // Filter on additive changes
-            })
-            .every((change) => {
-              return change.content.match(/"contributors":/)
-            })
-        })
-      })
-      .then((contributorsChanges) => {
-        if (contributorsChanges) {
-          const message =
-            'Do not make changes to package.json around contributors.'
-          const idea =
-            'This project only uses .all-contributorsrc for tracking contributors.'
-          fail(`${message} - <i>${idea}</i>`)
-        }
-      })
+// Check for any changes to the contributors section of package.json
+schedule(async () => {
+  if (!shouldRun) {
+    return;
   }
-}
+  const pd = await danger.git.JSONDiffForFile('package.json')
+
+  if (pd.contributors) {
```

Comment:
If contributors exists in the diff, then we fail. This is a better check than seeing if contributors exists in the text diff.